Dockerfile.local

```diff
@@ -0,0 +1,50 @@
+# =================================
```

Comment:
Why did you have to  introduce a new dockerfile for local testing. Have you tried the Dockerfile.dev instead? Dockerfile.dev would allow you to do live editing while testing without the need to rebuild the docker image after every change.

Reply:
I've tried it, but I have some issues with it, I wasn't finding reasons why, also I need to start it in "production", and your Dokerfile depends on Pypi, so I've wrote my own)

I can remove it from pull request, If you want)

Reply:
I'll add some comments on how to use Docker.dev. Then we can decide if there is still a need for another one.